modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Base64 encoded image was not displayed following `data:` text #TINY-8646
```

Comment:
This also wasn't restricted to images, it was base64 URIs in general and it was related to parsing not really how it was "rendered". The rendering was a side-effect of it not parsing correctly.